Source/Core/DolphinWX/ARCodeAddEdit.cpp

```diff
@@ -44,8 +44,8 @@ CARCodeAddEdit::CARCodeAddEdit(int _selection, const std::vector<ActionReplay::A
 	}
 	else
 	{
-		currentName = StrToWxStr(arCodes.at(selection).name);
-		tempEntries = arCodes.at(selection);
+		currentName = StrToWxStr(arCodes->at(selection).name);
+		tempEntries = arCodes->at(selection);
```

Comment:
That doesn't seem to work. I get errors like these:

    error C2039: 'name' : is not a member of 'std::vector<ActionReplay::ARCode,std::allocator<_Ty>>'